src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -104,11 +108,13 @@ def uploadWorker(workInput, results, dbsUrl, gzipEncoding=False):
                 logging.warning(msg)
                 results.put({'name': name, 'success': "error", 'error': msg})
             else:
-                reason = parseDBSException(exString)
-                msg = "Error trying to process block %s through DBS. Error: %s" % (name, reason)
-                logging.exception(msg)
+                logging.exception(dbsError.getReason())
```

Comment:
Please keep the message above, such that we know how exactly such messages are meant to be searched in the logs. So, I'd implement it like:
```
msg = f"Error trying to process block {name} through DBS. Reason: {dbsError.getReason()}"
logging.exception(dbsError.getReason())
results.put({'name': name, 'success': "error", 'error': msg})
```

and in the general exception below, I'd suggest something like:
```
        except Exception as ex:
            msg = f"Hit a general exception while inserting block {name. Error: {str(ex}}"
            logging.exception(msg)
            results.put({'name': name, 'success': "error", 'error': msg})
```

Reply:
done

Reply:
this comment is still pending. It was likely marked as resolved by mistake.